### DISCUSSION TITLE
SX1262 DutyCycledReceive question

### DISCUSSION THREAD

**Discussion by tshcherban** [jgromes/RadioLib#330] [Q&A]
Hi
Can someone help me with **_startReceiveDutyCycleAuto_**? I found in comments that _According to Semtech, receiver requires 8 symbols to reliably latch a preamble._ however i failed to find that info in a datasheet the only mention was for FSK modulation preamble length (im using SX1262, so used according pdf from Semtech's site). Just for experiments i run down to _minSymbols = 3_, which works quite unreliably, 4 - ~40% packets are missed, and _minSymbols = 5_ works good.
Later i found [this article](https://learn.circuit.rocks/esp32-lora-gateway-battery-optimized). It states that 2 symbols rx window works good.
Is working with _minSymbols < 8_ under specs and should be avoided?
**UPD**
also library sets radio awake for 2x minSymbols, but in the article above only for minSymbols amount

**Comment by jgromes** [comment]
This function was contributed by @BarryPSmith in https://github.com/jgromes/RadioLib/pull/84

Unfortunately I don't know where the information about 8 symbols being required comes from.

**Reply by BarryPSmith** [reply]
I believe the 'according to Semtech' was in response to a question I posed on their developer forum. Unfortunately they have restructured their forums and so the original post is no longer available. I searched through my emails, and found this that I received (I think that the responder was Seb @ Semtech. Emphasis added by me):

> Hi Barry,
> 
> I’ll try and explain the concepts:
> 
> - Preamble length: all sensitivity tests are run with programmed 8 symbols
> of Preamble, which in turns is 10.25 symbols if you add the built-in
> additional 4.25 symbols signalling the end of preamble. This explains the
> discrepancy you saw. **The demodulator works happily with this size, and down
> to overall 8.25 symbols, and we don’t recommend using a lower value. This
> being said, correlation may work with 6 symbols, but the probability of
> latching on to preamble with decrease dramatically, as you have observed,
> and will depend on when the demodulator actually start**s
> - Preamble length setting on the receiver: there is a built-in timeout and
> restart, whose duration (in symbols) is set by the preamble length setting
> in the receiver. Say your system is synchronous, and uses short (8 symbols)
> preamble; then assume that another system on the same channel/SF/BW is being
> heard by one of your receivers, and its preamble length is 10000 symbols.
> The receiver will latch onto the first 8 symbols of Preamble, and not find
> the end of preamble delimiter. It will timeout and restart its search,
> resetting its accrued frequency and time error estimations. This is the aim
> of setting the preamble length to a size equal or greater than the size of
> the transmitted preamble of the expected message. If you don’t know, and
> have a variety of sizes incoming, set it to the largest value.
> - RxDutyCycle: that is a good approach, use a long preamble in Tx and latch
> onto it in the receiver.
> 
> Best regards

(I can't remember the exact question I asked to get that response.)

Asides:
 - startReceiveDutyCycleAuto is supposed to err in favour of catching all the packets rather than the absolute minimum possible power consumption.
 - Can you point me to where in the linked article they say that 2 symbols works well? I find parts saying that they set it to two symbols, but don't see where they justify that.
 - My network uses startReceiveDutyCycleAuto for end nodes, and unfortunately they still sometimes miss packets. While they don't need 100% reliability, I find this a little frustrating because as far as I can tell (using a scope to look at current consumption) the radios are asleep and awake for periods that should ensure they never miss packets. I do use a preamble length of 384 symbols though, which might be an unusual use case.
